helloworld/terraform.tfvars

```diff
@@ -0,0 +1,10 @@
+project_id                  = "fellowship-test-internal"
+code_path                   = "/home/briansuk/code/ingest_test"
```

Comment:
Does each person need to download the code and update this to their local directory? Would it be better to set this relative to the project directory structure?

Reply:
Yeah, mentioned it in the comment but the TFVARS file shouldn't really be shared and everyone has their own, but I included mine just as an example, and because there's nothing sensitive in mine. I've also went into a little more information here: https://docs.google.com/document/d/1kTzjoOchPcduOMJxTt7XBWxMcP8gHC_bNAXGEi4x-ok/edit#heading=h.itgrk4vz6qn2

Reply:
I see, thanks!